frontend-v2/src/app/api.ts

```diff
@@ -176,7 +174,10 @@ export const api = backendApi.enhanceEndpoints({
       providesTags: (result, error, { id }) => [{ type: "Variable", id }],
     },
     variableUpdate: {
-      invalidatesTags: (result, error, { id }) => [{ type: "Variable", id }],
+      invalidatesTags: (result, error, { id }) => [
+        { type: "Variable", id },
+        { type: "Variable", id: "LIST" },
+      ],
```

Comment:
Prettier added a bunch of whitespace changes, but this is the actual code change in this PR.